OpenRA.Game/Network/Session.cs

```diff
@@ -90,11 +90,16 @@ public string FirstEmptyBotSlot()
 			return Slots.FirstOrDefault(s => !s.Value.Closed && ClientInSlot(s.Key) == null && s.Value.AllowBots).Key;
 		}
 
-		public bool IsSinglePlayer
+		public bool IsSingleHuman
 		{
 			get { return Clients.Count(c => c.Bot == null) == 1; }
 		}
 
+		public IEnumerable<Session.Client> HumanPlayers
```

Comment:
There is already another HumanPlayers property https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Game/GameInformation.cs#L38. (non bot players). Do you wish to rename both or only this new?